### PR TITLE
chore(CHANGELOG): add an entry for redis 7.2.4-1 release

### PR DESCRIPTION
**Pull request by SCedricThomas**

### CODE REVIEW

File: src/changelog/databases/_posts/2024-04-23-redis.7.2.4-1.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+modified_at: 2021-07-27 12:00:00
```
Comment (Frzk) [inline src/changelog/databases/_posts/2024-04-23-redis.7.2.4-1.md:2]:
```suggestion
modified_at: 2024-04-23 12:00:00
```

**Review by Frzk — CHANGES_REQUESTED** [review]

File: src/changelog/databases/_posts/2024-04-23-redis.7.2.4-1.md (hunk 1 above)
Comment (SCedricThomas) [reply src/changelog/databases/_posts/2024-04-23-redis.7.2.4-1.md:2]: arf missed that one

Comment (Frzk) [reply src/changelog/databases/_posts/2024-04-23-redis.7.2.4-1.md:2]: We've all been there ^^

**Review by Frzk — APPROVED** [review]
LGTM

**Pull request merged**